.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +11,19 @@
 - [ ] Tests added where applicable
 - [ ] UI changes look good on mobile
 - [ ] UI changes meet accessibility criteria
+
+### Merge instructions
+
+We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:
+
+```
+<type>[optional scope]: <description>
+```
+
+and contain the following structural elements:
+
+- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
+- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
+- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
+- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
```

Comment:
The intention is to have documentation easily accessibly for squash time commit messages, especially with these examples of the other commit types.